src/tiered/tiered_cursor.c

```diff
@@ -1118,7 +1180,7 @@ __wt_curtiered_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner,
 
     /* Check whether the exclusive open for a bulk load succeeded. */
     if (bulk && ret == EBUSY)
-        WT_ERR_MSG(session, EINVAL, "bulk-load is only supported on newly created trees");
+        ret = EINVAL;
```

Comment:
This error message was copied from the LSM code. But the python tests expect no message on this failure.

Reply:
This is kind of sad. We definitely would prefer to give the user more information than less. Is there any way to change python's expectations instead? @ddanderson any thoughts?

Reply:
We do return an error message (which the python test expects) in the similar case where an application tries to open a bulk cursor on a non-empty table. So we could certainly do the same in this case.

Of course, then we should be returning that message from the other cursor types as well, which would be the real work.    I'll create a new ticket for that.

Reply:
You can write tests to "expect" to see an error message in Python code.  Something like:
```
               errmsg = '/.*bulk-load is only supported.*/'
                self.assertRaisesWithMessage(wiredtiger.WiredTigerError,
                    lambda: self.session.open_cursor(uri, config), errmsg)
```